packages/wonder-blocks-toolbar/src/components/__docs__/toolbar.argtypes.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import * as React from "react";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Link from "@khanacademy/wonder-blocks-link";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+type Mappings = {[key: string]: React.Node};
+
+export const leftContentMappings: Mappings = {
+    none: null,
+    dismissButton: <IconButton icon={icons.dismiss} kind="tertiary" />,
+    hintButton: <IconButton icon={icons.hint} kind="primary" />,
+    multipleContent: (
+        <>
+            <IconButton icon={icons.zoomOut} kind="primary" />
+            <Strut size={Spacing.medium_16} />
+            <IconButton icon={icons.zoomIn} kind="primary" />
+        </>
+    ),
+};
+
+const buttonStyle = {width: 160};
+
+export const rightContentMappings: Mappings = {
+    none: null,
+    primaryButton: <Button>Submit</Button>,
+    tertiaryButton: <Button kind="tertiary">Import...</Button>,
+    nextVideoButton: <Button>Next Video</Button>,
```

Comment:
Do you think you could add an example with a light button for the dark background case?

Reply:
Good idea! I'll add it to the "dark" example.